erica_app/tests/elster_xml/test_elster_xml_generator.py

```diff
@@ -976,12 +980,13 @@ def test_add_vast_xml_nutzdaten_header(self):
     @freeze_time("2021-06-24")
     def test_add_vast_antrag_nutzdaten(self):
         xml_top = Element('top')
-        _add_vast_request_xml_nutzdaten(xml_top, self.valid_user_data)
+        with patch('erica.elster_xml.elster_xml_generator.dt.date.today', MagicMock(return_value=datetime.date(_BEANTRAGUNGSJAHR, 1, 1))):
+            _add_vast_request_xml_nutzdaten(xml_top, self.valid_user_data)
         xml_string = _pretty(xml_top)
 
         self.assertEqual('false', xml_top.find('.//SpezRechtAntrag/Veranlagungszeitraum/Unbeschraenkt').text)
-        self.assertEqual('2020', xml_top.find('.//SpezRechtAntrag/Veranlagungszeitraum/Veranlagungsjahre/Jahr').text)
-        self.assertEqual('2021-12-31', xml_top.find('.//SpezRechtAntrag/GueltigBis').text)
+        self.assertEqual(str(VERANLAGUNGSJAHR), xml_top.find('.//SpezRechtAntrag/Veranlagungszeitraum/Veranlagungsjahre/Jahr').text)
+        self.assertEqual('2022-12-31', xml_top.find('.//SpezRechtAntrag/GueltigBis').text)
```

Comment:
Were you meaning to use the `_BEANTRAGUNGSJAHR` in this test instead of 2022?

Reply:
Good catch 👍